addons/website_blog/migrations/9.0.1.0/post-migration.py

```diff
@@ -27,11 +27,10 @@ def convert_blog_post_cover(env):
         post = post_obj.browse(row[0])
         post.cover_properties = (
             '{{"background-image": "url({})",'
-            ' "opacity": "1",'
+            ' "opacity": "0",'
             ' "background-color": "oe_none",'
             ' "resize_class": "cover cover_full"}}'
         ).format(row[1])
-        break
```

Comment:
Why I put this `break`??!! Maybe a debug trick...? I'm crazy :stuck_out_tongue_closed_eyes: